src/AppInstallerCLICore/Workflows/SourceFlow.cpp

```diff
@@ -3,6 +3,7 @@
 #include "pch.h"
 #include "Resources.h"
 #include "SourceFlow.h"
+#include "PromptFlow.h"

```

Comment:
Seems like a leftover from a previous design?

Reply:
No, this is needed. The tasks for prompting for source agreements are used here; they were in WorkflowBase but I moved them to PromptFlow to have all the handling of interactivity in one place.